src/components/DashStat.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Grid from '@material-ui/core/Grid';
 import { withStyles } from '@material-ui/core/styles';
+import CircularProgress from './common/CircularProgress';
```

Comment:
There's no need to create a custom `CircularProgress` component.
The component we use elsewhere in the Admin Panel is part of the Material UI library: https://v4.mui.com/components/progress/#circular

Reply:
I have implemented the request changes!